docs/nydus-fscache.md

```diff
@@ -0,0 +1,115 @@
+# Nydus EROFS fscache user guide
+
+This guide shows you how to use fscache-based EROFS nydus image service to launch docker containers with the fscache-enabled in-kernel erofs on-demand download feature.
+
+**Please be careful**, currently, the user-space daemon only implements _the basic functionality_ and it's aimed to test the fscache on-demand kernel code as a real end-to-end workload for container use cases, so it may take more extra steps compared with existing well-done solutions. This guide can be _frequently updated_ due to the overall implementation changes, so please make sure that you're now referring to the latest document version.
+
+Currently only docker graphdriver is supported since it's much easier to integrate than containerd nydus-snapshotter, we will complete nydus-snapshotter together with the fscache on-demand download upstream process. You could help us if you're also interested in it, so much appreciated!
+
+## Prepare the kernel
+
+Be aware of using the fscache-enabled erofs linux kernel, it can be built with the following steps:
+
+1.  ``git clone git://git.kernel.org/pub/scm/linux/kernel/git/xiang/erofs.git -b jeffle/fscache`` \
```

Comment:
please help update it as ``git clone git://git.kernel.org/pub/scm/linux/kernel/git/xiang/erofs.git``
drop ``-b jeffle/fscache``